simulator/src/main/java/com/hedera/block/simulator/grpc/PublishStreamGrpcClientImpl.java

```diff
@@ -52,14 +54,17 @@ public class PublishStreamGrpcClientImpl implements PublishStreamGrpcClient {
     private final AtomicBoolean streamEnabled;
     private ManagedChannel channel;
     private final MetricsService metricsService;
+    private final List<String> lastKnownStatuses = new ArrayList<>();
+    private int publishedBlocks;
```

Comment:
Rather than use a new variable (publishedBlocks), could we use a new block counter metric in `SimulatorMetricMetadata` like `LiveBlocksSent("live_blocks_sent", "Live Blocks Sent")`? You can query it like:
```
@Override
    public long getPublishedBlocks() {
        return metricsService.get(LiveBlockItemsSent).get();
    }
```
Later, we'll be able to see it in a dashboard as well.

Reply:
Done, thanks for this. ✅